tensorflow/core/common_runtime/executor.cc

```diff
@@ -1792,6 +1796,47 @@ void ExecutorState::Process(TaggedNode tagged_node, int64 scheduled_nsec) {
     Entry* first_input = input_tensors + item.input_start;
     outputs.clear();
 
+    string msg;
```

Comment:
Extract the construction of this msg into a separate function.

Reply:
Applied in commit d0857f8c16d48d6a28e8be2cc098e6666090ee34